test/e2e/page-objects/pages/test-dapp.ts

```diff
@@ -321,6 +321,26 @@ class TestDapp {
     });
   }
 
+  /**
+   * Verifies the accounts connected to the test dapp.
+   *
+   * @param connectedAccounts - The expected connected accounts separated by a comma. If no accounts are connected we can omit the param.
+   */
+  async check_connectedAccounts(connectedAccounts: string = '') {
+    console.log('Verify connected accounts');
+    if (connectedAccounts) {
+      await this.driver.waitForSelector({
+        css: '#accounts',
```

Comment:
could we use the variable defined here? https://github.com/MetaMask/metamask-extension/blob/0e10bab6bc28bb83916c14426fa0439f69903418/test/e2e/page-objects/pages/test-dapp.ts#L35

Reply:
oh absolutely! changed [here](https://github.com/MetaMask/metamask-extension/pull/29273/commits/90dfcf15ac619427f73833ef3d1b7a965978174d), thank you!